iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ValidationUtil.java

```diff
@@ -66,33 +123,58 @@ public static void checkActivePowerLimits(Validable validable, double minP, doub
         }
     }
 
-    public static void checkTargetDeadband(Validable validable, String validableType, boolean regulating, double targetDeadband) {
+    public static ValidationLevel checkTargetDeadband(Validable validable, String validableType, boolean regulating, double targetDeadband, boolean throwException) {
+        return checkTargetDeadband(validable, validableType, regulating, targetDeadband, throwException, Reporter.NO_OP);
+    }
+
+    public static ValidationLevel checkTargetDeadband(Validable validable, String validableType, boolean regulating, double targetDeadband, boolean throwException, Reporter reporter) {
+        ValidationLevel validationLevel = ValidationLevel.LOADFLOW;
         if (regulating && Double.isNaN(targetDeadband)) {
-            throw new ValidationException(validable, "Undefined value for target deadband of regulating " + validableType);
+            throwExceptionOrLogError(validable, "Undefined value for target deadband of regulating " + validableType, throwException, reporter);
+            validationLevel = ValidationLevel.min(validationLevel, ValidationLevel.STATE_ESTIMATION);
         }
         if (targetDeadband < 0) {
-            throw new ValidationException(validable, "Unexpected value for target deadband of " + validableType + ": " + targetDeadband + " < 0");
+            throwExceptionOrLogError(validable, "Unexpected value for target deadband of " + validableType + ": " + targetDeadband + " < 0", throwException, reporter);
+            validationLevel = ValidationLevel.min(validationLevel, ValidationLevel.STATE_ESTIMATION);
```

Comment:
Same comment as before, it should be forbidden to have a target deadband negative, for both level.